rllib/algorithms/algorithm_config.py

```diff
@@ -889,9 +889,7 @@ def evaluation(
             else:
                 self.evaluation_config = evaluation_config
         if off_policy_estimation_methods is not None:
-            self.evaluation_config[
-                "off_policy_estimation_methods"
-            ] = off_policy_estimation_methods
+            self.off_policy_estimation_methods = off_policy_estimation_methods
```

Comment:
I'm not sure how we could type this, because the OPE dict is more like `Dict[str, Dict[str, Any]]` where the OPE type is passed in under the "type" key e.g. 
`"dm_fqe": {
                        "type": DirectMethod,
                        "q_model_config": {"type": FQETorchModel, "n_iters": n_iters},
                    },`

I can create config objects for OPE in a future PR. We should also change this `"config": {"type": ...}` pattern across RLlib like @avnishn mentioned; I used it here for conformity.

Reply:
Done